tfs/src/main/java/hudson/plugins/tfs/model/BuildCommand.java

```diff
@@ -209,23 +209,27 @@ static void contributeTeamBuildParameterActions(final Map<String, String> teamBu
             final boolean isTeamGit = "TfGit".equalsIgnoreCase(provider)
                     || "TfsGit".equalsIgnoreCase(provider);
             if (isTeamGit) {
-                final String collectionUriString = teamBuildParameters.get(SYSTEM_TEAM_FOUNDATION_COLLECTION_URI);
-                final URI collectionUri = URI.create(collectionUriString);
+            	// "Build.Repository.Uri" is null/whitespace/empty when the 'Jenkins Queue Job' task runs in TFS Release Management.
+            	// In this case, do not reference a CommitParameterAction in the actions reported as unsupported.
                 final String repoUriString = teamBuildParameters.get(BUILD_REPOSITORY_URI);
-                final URI repoUri = URI.create(repoUriString);
-                final String projectId = teamBuildParameters.get(SYSTEM_TEAM_PROJECT);
-                final String repoId = teamBuildParameters.get(BUILD_REPOSITORY_NAME);
-                final String commit = teamBuildParameters.get(BUILD_SOURCE_VERSION);
-                final String pushedBy = teamBuildParameters.get(BUILD_REQUESTED_FOR);
-                final GitCodePushedEventArgs args = new GitCodePushedEventArgs();
-                args.collectionUri = collectionUri;
-                args.repoUri = repoUri;
-                args.projectId = projectId;
-                args.repoId = repoId;
-                args.commit = commit;
-                args.pushedBy = pushedBy;
-                final CommitParameterAction action = new CommitParameterAction(args);
-                actions.add(action);
+                if (repoUriString != null && !repoUriString.trim().isEmpty()) {

```

Comment:
Is `Build.Repository.Uri` present in the JSON payload and has an empty/null value or is it not present at all?  For example, [line 207](https://github.com/jenkinsci/tfs-plugin/pull/142/commits/465021e4c1bcbab09fb61022c2fd539515771988#diff-f7db47497a32ee414942dd7079038aa5R207) above checks if the map contains an entry with what resolves to the `Build.Repository.Provider` key.

Otherwise, might I recommend using `org.apache.commons.lang.StringUtils.isNotBlank()`?